### PR TITLE
Cnft1 1608 [Patient Design] - Profile - Font size for all tables should be reduced to 14px

### PR DESCRIPTION
**Pull request by mdavis-enquizit**
## Description

[Patient Design] - Profile - Font size for all tables should be reduced to 14px
[Patient Design] - Profile - General Patient Information - Borders go edge to edge, same light grey (not black)
[Patient Design] - Profile - Success message - text should stay left aligned instead of centered
## Tickets

* [CNFT1-1608](https://cdc-nbs.atlassian.net/browse/CNFT1-1608)
Screenshot 2023-11-27 at 11 48 53 AM
Screenshot 2023-11-27 at 12 15 05 PM

* [CNFT1-1606](https://cdc-nbs.atlassian.net/browse/CNFT1-1606)
* 
Screenshot 2023-11-27 at 11 50 26 AM
Screenshot 2023-11-27 at 12 16 02 PM

* [CNFT1-1609](https://cdc-nbs.atlassian.net/browse/CNFT1-1609)
* 
Screenshot 2023-11-28 at 1 22 15 PM
Screenshot 2023-11-28 at 1 20 57 PM

## Steps to verify

1. Login and view UI changes on patient page

[CNFT1-1608]: https://cdc-nbs.atlassian.net/browse/CNFT1-1608?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[CNFT1-1606]: https://cdc-nbs.atlassian.net/browse/CNFT1-1606?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[CNFT1-1609]: https://cdc-nbs.atlassian.net/browse/CNFT1-1609?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/modernization-ui/src/components/Table/style.scss [hunk 1]
```diff
@@ -14,8 +14,8 @@
     }
 }
 
-td {
-    font-size: 16px !important;
+td, th {
+    font-size: 14px !important;
```
Comment (hclarkEnq) [inline apps/modernization-ui/src/components/Table/style.scss:18]: can this be changed to rem?

**Review by hclarkEnq — DISMISSED** [review]
👍

**Review by hclarkEnq — APPROVED** [review]

File: apps/modernization-ui/src/components/Table/style.scss [hunk 2]
```diff
@@ -13,6 +13,15 @@
         }
     }
 }
+
+td, th {
+    font-size: 14px !important;
+}
+
+.show-length-text {
```
[the remark this answers is not in the document]
Comment (mdavis-enquizit) [reply apps/modernization-ui/src/components/Table/style.scss:21]: Merge conflict took away the  } on line 19.

**Review by adamloup-enquizit — APPROVED** [review]
Looks good!  Sorry for the delay in response.

**Review by hclarkEnq — APPROVED** [review]

**Pull request merged**